### PR TITLE
Fixed ordering assumptions in server_streaming interop test

### LINKED ISSUE #1720: Node flake in interop test
-- deleted bad log

### PR DESCRIPTION
**Pull request by murgatroid99**
This fixes #1720

### CODE REVIEW

File: src/node/src/client.js [hunk 1]
```diff
@@ -81,7 +81,7 @@ function _write(chunk, encoding, callback) {
   batch[grpc.opType.SEND_MESSAGE] = this.serialize(chunk);
   this.call.startBatch(batch, function(err, event) {
     if (err) {
```
Comment (ctiller) [inline src/node/src/client.js]: Does this collapse to simply callback(err) then?

Comment (murgatroid99) [inline src/node/src/client.js]: It looks like it. There's also a failure on Travis that may or may not be related to what I did, but I can't reproduce it in over 1000 runs on my local machine.

**Comment by murgatroid99** [comment]
OK, I think that change to `_write` in `client.js` is causing problems, and it shouldn't actually be relevant. I'm reverting it.

**Comment by murgatroid99** [comment]
green

**Pull request merged**